components/content-service/pkg/layer/provider.go

```diff
@@ -287,8 +295,12 @@ func (s *Provider) GetContentLayerPVC(ctx context.Context, owner, workspaceID st
 			if err != nil {
 				return nil, nil, err
 			}
+			layerReady, err := workspaceReadyLayerPVC(csapi.WorkspaceInitFromOther)
```

Comment:
To be honest, I don't know what is the correct `WorkspaceInitSource` for the snapshot.

Reply:
I test the snapshot without the PVC feature flag. The content within `read` file is `{"source":"from-backup"}`.

Reply:
I don't think it matters that much if it is from backup or from other, but for consistency, updated to match. :heart: